site/src/components/gradients.md

```diff
@@ -98,3 +100,25 @@ $border-gradient-direction: (
 	'top': 'block-start' // an additional direction
 ) !default;
 ```
+
+## Gradient card
```

Comment:
I don't feel too strongly about this but I would recommend moving this to a separate documentation page. This one is getting quite long. We probably should have done this with the text color gradients too ...